sova/supervisor/planner.py

```
@@ -358,6 +360,85 @@ async def _get_recent_failures(self) -> str:
+                "| Issue | Failed | Succeeded | Cost | Last Error |",
+                "|-------|--------|-----------|------|------------|",
+            ]
+            for issue_num in sorted(issues_with_runs, key=lambda x: int(x)):
+                health = health_by_issue[issue_num]
+                error = health["last_error"] or "none"
+                cost = cost_by_issue.get(issue_num, Decimal(0))
+                failed = health["failed"]
+                succeeded = health["succeeded"]
+                lines.append(f"| #{issue_num} | {failed} failed | {succeeded} succeeded | ${cost:.2f} | {error} |")
+            return "\n".join(lines)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)